src/auth/auth_err_test.go

```diff
@@ -1,88 +0,0 @@
-package auth
-
-import (
-	"testing"
-
-	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
-	"github.com/pachyderm/pachyderm/v2/src/internal/require"
-)
-
-// grpcify returns an error e such that e.Error() is similar to what grpc
-// errors emit (though this doesn't convert 'error' to an actual GRPC error)
-func grpcify(err error) error {
-	return errors.Errorf("rpc error: code = Unknown desc = %s", err.Error())
-}
-
-func TestIsErrNotActivated(t *testing.T) {
```

Comment:
Just curious, why delete these tests? They are kind of pedantic—is it that, or do they no longer work?

Reply:
Re-added these, I ripped them out to get things compiling and forgot to circle back to update them.